testsuite/MDAnalysisTests/dummy.py

```diff
@@ -0,0 +1,210 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2017 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+"""Mock Universe and Topology generated from scratch with default values
+
+"""
+from __future__ import absolute_import, division
+
+from six.moves import range
+
+import numpy as np
+import string
+import itertools
+
+
+# Elements  of the standard mock Universe
+# 5 atoms per residues, 5 residues per segment
+_N_ATOMS = 125
+_N_RESIDUES = 25
+_N_SEGMENTS = 5
+_ATOMS_PER_RES = _N_ATOMS // _N_RESIDUES
+_RESIDUES_PER_SEG = _N_RESIDUES // _N_SEGMENTS
+
+
+def make_Universe(extras=None, size=None,
+                  trajectory=False, velocities=False, forces=False):
+    """Make a dummy reference Universe
+
+    Allows the construction of arbitrary-sized Universes. Suitable for
+    the generation of structures for output.
+
+    Preferable for testing core components because:
+      * minimises dependencies within the package
+      * very fast compared to a "real" Universe
+
+    Parameters
+    ----------
+    extras : list of strings
+      extra attributes to add to Universe:
+      u = make_Universe(('masses', 'charges'))
+      Creates a lightweight Universe with only masses and charges.
+    size : tuple of int
```

Comment:
Could perhaps mention default values for the various parameters in the docstring, though probably not crucial.

Reply:
done